aws/src/database/dynamodb/scripts/patches_and_areas_table/clean_patches_and_areas.py

```diff
@@ -0,0 +1,89 @@
+from dataclasses import asdict
+
+from mypy_boto3_dynamodb.service_resource import Table
+
+from aws.src.database.domain.dynamo_domain_objects import ResponsibleEntity, ResponsibleEntityContactDetails
+from aws.src.database.dynamodb.utils.get_dynamodb_table import get_dynamodb_table
+from enums.enums import Stage
+from aws.src.database.domain.dynamo_domain_objects import Patch
+
+from faker import Faker
+
+
+def link_patches_to_areas(table: Table):
+    """
+    Set patch ParentIds to the correct area and update the PatchesAndAreas table
+    """
+    patches_raw = table.scan()["Items"]
+    patches = [Patch.from_data(patch) for patch in patches_raw]
+
+    areas = [item for item in patches if item.patchType == "area" and item.name != "Hackney"]
+    assert len(areas) == 7, [area.name for area in areas]
+    patches = [item for item in patches if item.patchType == "patch"]
+    assert len(patches) >= 46, ([patch.name for patch in patches], len(patches))
```

Comment:
Could you explain what these length assertions do?

Reply:
Just sanity checks. There are 7 areas and 46 patches in the database on prod